docs/source/conf.py

```diff
@@ -80,6 +80,12 @@
 #  "**": [],
 #}
 
+# Ignore links that do not work with github actions link checking
+# https://github.com/neuroinformatics-unit/actions/pull/24#issue-1978966182
+linkcheck_ignore  = [
```

Comment:
I think you would still want to check that the base URL works, and only skip checking the anchors, rather than ignoring individual full URLs with `linkcheck_ignore`?: 
```suggestion
linkcheck_anchors_ignore_for_url = [
    "https://neuroinformatics.zulipchat.com/",
]
```